fix email verification

Following the email verification link sent to my email results in this error:

https://accounts-latest.dev.lcip.org/v1/verify_email?uid=db95f7a03a1b4b30b09580ab29e18b1e&code=feddf9aa07390cf091ef9191f9dfc3d9

```
Error: Cannot find module 'html'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at new View (/home/app/code/node_modules/express/lib/view.js:43:49)
    at Function.app.render (/home/app/code/node_modules/express/lib/application.js:487:12)
    at ServerResponse.res.render (/home/app/code/node_modules/express/lib/response.js:756:7)
    at routeToVerify (/home/app/code/server/lib/routes.js:10:9)
    at /home/app/code/server/lib/routes.js:23:5
    at callbacks (/home/app/code/node_modules/express/lib/router/index.js:161:37)
```

--- a/app/scripts/router.js
+++ b/app/scripts/router.js
@@ -35,7 +35,7 @@ function ($, Backbone, IntroView, SignInView, SignUpView, ConfirmView, SettingsV
       'birthday': 'showBirthday',
       'create_account': 'showCreateAccount',
       'cannot_create_account': 'showCannotCreateAccount',
-      'complete_sign_up': 'showCompleteSignUp'
+      'verify_email': 'showCompleteSignUp'
     },
 
     initialize: function () {

--- a/server/lib/routes.js
+++ b/server/lib/routes.js
@@ -6,43 +6,10 @@ const config = require('./configuration');
 const path = require('path');
 
 module.exports = function(app) {
-  function routeToVerify(req, res) {
-    res.render('verify_email.html', {
-      fxa: config.get('fxaccount_url'),
-      code: req.query.code,
-      uid: req.query.uid
-    });
-  }
-
-  // route both of these to 'verify_email'
-  app.get('/verify_email', function(req, res) {
-    routeToVerify(req, res);
-  });
-
+  // handle email verification links
   app.get('/v1/verify_email', function(req, res) {
-    routeToVerify(req, res);
+    res.redirect(req.originalUrl.slice(3));
   });
-
-  app.get('/flow/:page?',
-    function(req, res) {
-      res.render('accounts/desktop_flow.html', {
-        fxa: config.get('fxaccount_url'),
-        user: "null",
-        verified: 0,
-        page: JSON.stringify(req.params.page || ''),
-        desktop: true
-      });
-    });
-
-  app.get('/mobile/:page?',
-    function(req, res) {
-      res.render('accounts/flow.html', {
-        fxa: config.get('fxaccount_url'),
-        user: "null",
-        verified: 0,
-        page: JSON.stringify(req.params.page || '')
-      });
-    });
 
   app.get(/\/[^.]*$/, function(req, res, next) {
     // setting the url to / will use the correct index.html for either dev or

--- a/tests/functional/complete_sign_up.js
+++ b/tests/functional/complete_sign_up.js
@@ -12,7 +12,7 @@ define([
   'use strict';
 
   var AUTH_SERVER_ROOT = 'http://127.0.0.1:9000/v1';
-  var PAGE_URL_ROOT = 'http://localhost:3030/complete_sign_up';
+  var PAGE_URL_ROOT = 'http://localhost:3030/verify_email';
   var PASSWORD = 'password';
   var email;
 

